dask/dataframe/tests/test_dataframe.py

```diff
@@ -300,13 +300,33 @@ def test_describe():
     assert ds.describe(split_every=2)._name != ds.describe()._name
     assert ddf.describe(split_every=2)._name != ddf.describe()._name
 
+    s = pd.Series(list(range(10)) * 6)
```

Comment:
General comment for this whole file: it would be good to use `pytest.mark.parametrize` here to minimize duplicating code.